data/json/mutation_category.json

```diff
@@ -0,0 +1,155 @@
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_cat",
+		"name" : "Feline",
+		"category" : "FELINE",
+		"mutagen_message" : "As you lap up the last of the mutagen, you wonder why...",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_wolf",
+		"name" : "Wolf",
+		"category" : "LUPINE",
+		"mutagen_message" : "You feel an urge to mark your territory. But then it passes.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_cattle",
+		"name" : "Cattle",
+		"category" : "CATTLE",
+		"mutagen_message" : "Your mind and body slow down. You feel peaceful.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_cephalopod",
+		"name" : "Cephalopod",
+		"category" : "CEPHALOPOD",
+		"mutagen_message" : "Your mind is overcome by images of eldritch horrors...and then they pass.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_bird",
+		"name" : "Bird",
+		"category" : "BIRD",
+		"mutagen_message" : "Your body lightens and you long for the sky.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_lizard",
+		"name" : "Lizard",
+		"category" : "TROGLOBITE",
+		"mutagen_message" : "You yearn for a cool, dark place to hide.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_alpha",
+		"name" : "Alpha","//" : "5-15 pain, 66% for each of the follow-ups, so slightly better odds (designed for injection).",
+		"category" : "ALPHA",
+		"mutagen_message" : "You feel...better. Somehow.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_medical",
+		"name" : "Medical", 
+		"category" : "MEDICAL",
+		"mutagen_message" : "You can feel the blood rushing through your veins and a strange, medicated feeling washes over your senses.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_chimera",
+		"name" : "Chimera",
+		"category" : "CHIMERA",
+		"mutagen_message" : "You need to roar, bask, bite, and flap.  NOW.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_elf",
+		"name" : "Elf",
+		"category" : "ELFA",
+		"mutagen_message" : "Nature is becoming one with you...",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
+	},{
+		"type" : "mutation_category",
+		"id" : "mutation_raptor",
+		"name" : "Raptor",
+		"category" : "RAPTOR",
+		"mutagen_message" : "Mmm...sweet, bloody flavor...tastes like victory.",
+		"iv_message" : "",		
+		"memorial_message" : "NULL"		
```

Comment:
I haven't worked my way to it yet, but the strings aren't optional, I tested it, and the game will throw a json error if someone tries to create a category without these base strings.